src/main/java/com/teamcautionrobotics/robot/Robot.java

```diff
@@ -37,16 +37,16 @@
      * 
      * Driver controls:
      * 
-     * Left joystick: X axis, robot turn control; Button 1, Jack for HAB;
-     * Button 2, Toggle aiming lights
+     * Left joystick: X axis, robot turn control; Button 1, Jack for HAB; Button 2,
+     * Toggle aiming lights
      * 
      * Right Joystick: Y axis, robot forward and backward control; Button 2, smooth
      * deriving toggle; Button 3, precision turning mode
```

Comment:
Driver right 2, driver right 3, and manipulator B control the manipulator grabber.